guides/callbacks/callbacks.md

```diff
@@ -0,0 +1,22 @@
+# Bandwidth Callbacks
+
+## Bandwidth Callbacks Overview
+
+Bandwidth's APIs operate on a system of callbacks. Most API requests send callbacks, and notices of incoming events (phone call, text message) are sent via callbacks as well.
+
+## How To Utilize Callbacks
+In order to utilize callbacks, you must run a server with a URL that is expecting to receive callbacks from Bandwidth. This URL must be provided to Bandwidth through your Bandwidth application settings, or as part of your API request. For voice callbacks, this endpoint must return valid BXML for Bandwidth to use. More information on Bandwidth applications can be found [here](../../applications/about.md).
+
+Callback URLs can (and ideally should) be password protected. To allow Bandwidth to access these URLs, the callback URL must be defined to follow the format `https://{username}:{password}@yourdomain.com/`.
```

Comment:
This is no longer the case, this is managed within the application

Reply:
```
 To allow Bandwidth to access these URLs, applications in the Bandwidth Dashboard can be configured with a username and password used to authenticate on your callback URL.
```

Would this be sufficient, or should it also link out to an example?